### DISCUSSION TITLE
build from source for cuda-toolchain error?how to solve the problem

### DISCUSSION THREAD

**Discussion by wangzy0327** [intel/llvm#6255] [Q&A]
**Describe the bug**
I exactly build clang toolchain on CUDA compiler dependency on this link:
https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#Build DPC++ toolchain with support for NVIDIA CUDA

my source project version is: 
https://github.com/intel/llvm/releases?page=19
[oneAPI DPC++ Compiler 2021-09](https://github.com/intel/llvm/releases/tag/2021-09)

**To Reproduce**
Please describe the steps to reproduce the behavior:

1. python3 $DPCPP_HOME/intel-llvm/buildbot/configure.py  -t debug --cuda -o $DPCPP_HOME/build-cuda-debug
2. python3 $DPCPP_HOME/intel-llvm/buildbot/compile.py -o $DPCPP_HOME/build-cuda-debug -j 32

The error log as below:
FAILED: lib/libsycl.so.5.4.0-0

![image](https://user-images.githubusercontent.com/22990858/172099947-b7ec76c9-0dfb-456e-8be3-1d64fe5ae276.png)

**Environment (please complete the following information):**

- OS: Linux  (CentOS Linux release 7.9.2009 (Core))
- Target device and vendor: [NVIDIA GPU]
- DPC++ version: [e.g. commit hash or output of `clang++ --version`]
- Dependencies version: [e.g. low-level runtime versions (like NEO 20.04)]

**Additional context**

**Comment by zjin-lcf** [comment]
Does the build succeed on Ubuntu 20.04 ?
